Podfile

```diff
@@ -13,7 +13,7 @@ use_frameworks!
 def shared_pods
   
   # CruGlobal pods
-  pod 'GodtoolsToolParser', '0.6.0'
+  pod 'GodtoolsToolParser', '0.6.1-SNAPSHOT'
```

Comment:
What does `snapshot` mean?

Reply:
it's a concept I took from java maven versioning conventions. SNAPSHOT is a moving target for a build at the tip of the current development branch.

So, `SNAPSHOT` versions can change frequently, but they allow you to access new features before they end up in a stable (unchanging) release.

I try to make sure and create a stable release of the shared parser before we release iOS or Android apps to the store, that way we in theory could always build older versions of the app from git if we need to debug something that happens on a specific version of the app.

Reply:
https://stackoverflow.com/a/5901460/4721910